From 7ce70832ee84c9d3c491fedb15cf9abd2aacb5bf Mon Sep 17 00:00:00 2001
From: Alexander Miller <alex.miller@gmx.de>
Date: Sun, 26 Oct 2025 23:52:08 +0100
Subject: [PATCH] Increase minimum required CMake version to 3.12

Since version 3.31, CMake warns that compatibility with CMake < 3.10
will be removed from a future version of CMake. Moreover, the project
(conditionally) uses the FindPython module which was introduced only
in CMake version 3.12.
---
 CMakeLists.txt | 2 +-
 README.md      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8cbc12b..71d82d4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.12)
 
 project(trrntzip
   VERSION 1.3
diff --git a/README.md b/README.md
index c70c883..69933ea 100644
--- a/README.md
+++ b/README.md
@@ -18,7 +18,7 @@ comes from).
 
 * A C compiler (e.g. gcc or clang)
 * [zlib](http://zlib.net/) (at least version 1.2.2)
-* [CMake](https://cmake.org/) (at least version 3.5)
+* [CMake](https://cmake.org/) (at least version 3.12)
 
 ## Building
 
